assets/scripts/main.js

```diff
@@ -444,13 +444,21 @@ if (document.querySelector('main')) {
       element.setAttribute('title',      t('to footnote') + ' ' + element.textContent);
     });
 
+    let pageLang = document.documentElement.lang || document.getElementsByTagName('html')[0].getAttribute('xml:lang');
```

Comment:
![nits](https://img.shields.io/badge/review-nits-green.svg)

If this value is not going to be changed forever, I recommended to use `const` instead of `let`!